QuoteGeneration/pccs/container/Dockerfile

```diff
@@ -4,6 +4,9 @@ FROM ubuntu:23.04 AS builder
 # Define arguments used across multiple stages
 ARG DCAP_VERSION=DCAP_1.20
 ARG NODE_MAJOR=20
+ARG NODE_MINOR=11
+ARG NODE_PATCH=1
+ARG NODE_REVISION=1nodesource1
```

Comment:
perhaps add a comment here to say why we need to pin to a specific version (preferably with a link to the upstream issue)